build.gradle

```diff
@@ -128,7 +143,8 @@ checkstyle {
 
 pmd {
   toolVersion = "6.18.0"
-  sourceSets = [sourceSets.main, sourceSets.test, sourceSets.functionalTest, sourceSets.integrationTest, sourceSets.smokeTest]
+  //sourceSets = [sourceSets.main, sourceSets.test, sourceSets.functionalTest, sourceSets.integrationTest, sourceSets.smokeTest]
+  sourceSets = [sourceSets.test]
```

Comment:
I think, this is required for all the sources. Could you please put some comments here highlighting it needs to be enabled for all the sources later.

Reply:
Updated comments.